Automattic/jetpack
# Contact Form: align half-width fields on same row

Fixes https://github.com/Automattic/jetpack/issues/34567

## Proposed changes:

Make sure _Contact Form_ fields that are 50% wide or less are aligned on the same row for all form styles. It's currently only the case for the _Default_ style.

| Before | After |
|-|-|
|<img width="500" alt="Screenshot 2023-12-13 at 4 24 17 PM" src="https://github.com/Automattic/jetpack/assets/1620183/90d297d0-921c-44f8-b488-c58b840956e2">|<img width="500" alt="Screenshot 2023-12-13 at 4 44 41 PM" src="https://github.com/Automattic/jetpack/assets/1620183/ffe69153-df49-401d-bff0-1f6d7387927f">|

### Other information:



## Jetpack product discussion


Project thread: pf5801-87-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- Create a new post
- Add the _Contact Form_ block
- Make sure that two consecutive fields have a width of 50% or less
- Open the preview
- For each styles (_Default_, _Animated_, and _Outlined_), check that the fields are on the same row
